modules/evm/test/precompiles/erc20/index.test.ts

```diff
@@ -43,18 +48,71 @@ describe("ERC20", () => {
         ownerContract = new ethers.Contract(contractAddress, abi, ownerSigner);
         userContract = new ethers.Contract(contractAddress, abi, userSigner);
 
-        await executeTx(userContract.transfer(ownerSigner.address, erc20.feeFund));
-
         tokenAmount = toBigInt(erc20.amount);
     });
 
-    // Notice: This acts as a blockchain state reset by burning all tokens from the owner.
-    // Ensures that each test starts with a clean slate for the owner.
+    beforeEach(async () => {
+        await executeTx(userContract.transfer(ownerSigner.address, erc20.feeFund));
+    });
+
     afterEach(async () => {
-        await resetOwnerState(ownerContract, userContract, ownerSigner, userSigner);
+        if (chain.env === "localnet") {
```

Comment:
Estaria be que aquesta llogica es gestiones directament dins de la funcio `resetOwnerState` i aixi no cal repetirla en mes tests